aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8.md

```diff
@@ -81,7 +81,7 @@ The [complete sample code](https://github.com/dotnet/AspNetCore.Docs.Samples/tre
 
 Parameter binding for minimal APIs binds parameters through [dependency injection](xref:fundamentals/dependency-injection) when the type is configured as a service. It's not necessary to explicitly apply the [`[FromServices]`](xref:Microsoft.AspNetCore.Mvc.FromServicesAttribute) attribute to a parameter. In the following code, both actions return the time:
 
-[!code-csharp[](~/release-notes/aspnetcore-7/samples/ApiController/Program.cs?name=snippet_min)]
+:::code language="csharp" source="~/release-notes/aspnetcore-7/samples/ApiController/Program.cs" id="snippet_min" highlight="8-9":::
```

Comment:
Added highlights for the dependency injection:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/1f3f8f7d-6121-4961-ad12-f0a64ee5b98a)